apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx

```diff
@@ -161,42 +161,52 @@ export default class ModerationNotificationList extends Component {
     const gotoTopText = django.gettext('Go to top')
 
     return (
-      <div className="row mb-2">
-        <div className="col-12">
-          <h1>
-            <a href={projectUrl}>{projectTitle}</a>
-          </h1>
-          <span className="text-muted">
-            {byText}
-            {organisation}
-          </span>
-          <div
-            className="filter-bar__filled"
-            aria-label={django.gettext('Filter')}
-          >
+      <>
+        <div className="row mb-2">
+          <div className="col-12">
+            <h1>
+              <a href={projectUrl}>{projectTitle}</a>
+            </h1>
+            <span className="text-muted">
+              {byText}
+              {organisation}
+            </span>
+          </div>
+        </div>
+
+        <div
+          className="row"
+          aria-label={django.pgettext('kosmo', 'Filter')}
+        >
+          <div className="col-md">
             <Filter
-              filterClass="mt-3 mt-lg-5 me-lg-3 dropdown dropdown-menu-end"
+              filterClass="filter--full dropdown dropdown-menu-end"
```

Comment:
out of curiosity: is there a general rule we follow when deciding, when to use bs and when to use our own classes?

Reply:
@khamui that's a good question! the answer is kind of but not really, in this case the BS classes `dropdown dropdown-menu-end` are needed for the functionality so unless it is general styling that should be applied to every dropdown we should not add anything to that class so having an additional class is preferable. Also as in this case we will want all children of filter--full to be full width, we add a class as this is simpler then assigning bs classes in multiple places. I don't think it's particularly standard to have such a mix as we do and finding the right medium is pretty hard but as bootstrap can be very long winded to get something precise it's kind of if it 'feels' right then we go with bs but if not with a component class.